src/Raven.Server/Documents/Handlers/StreamCsvResultWriter.cs

```diff
@@ -21,6 +21,13 @@ public abstract class StreamCsvResultWriter<T> : IStreamQueryResultWriter<T>
         private (string, string)[] _properties;
         private bool _writeHeader = true;
 
+        private readonly HashSet<string> _metadataPropertiesToSkip = new HashSet<string>
```

Comment:
@grisha-kotler can we create some sort of fail check e.g. a test that will help us to manage those in the future? Currently this is a very easy place to skip when new metadata properties are being added e.g. timeseries.